packages/jest-runner/test/integration/jest-test-runner.it.spec.ts

```diff
@@ -44,8 +47,10 @@ describe(`${JestTestRunner.name} integration test`, () => {
       const runResult = await jestTestRunner.dryRun({ coverageAnalysis: 'off' });
 
       assertions.expectCompleted(runResult);
-      expect(runResult.tests[0].name).to.equal('Add should be able to add two numbers');
-      expect(runResult.tests[0].timeSpentMs).to.be.above(-1);
+      const result = runResult.tests.find((test) => test.id === 'Add should be able to add two numbers');
+      expect(result).to.not.be.null;
+      expect(result!.name).to.equal('Add should be able to add two numbers');
+      expect(result!.timeSpentMs).to.be.above(-1);
```

Comment:
I had to make this changes because I kept getting random errors when running the tests.

I figured that it was because by testing the first test in the array we were making assumptions about the order in which jest runs tests. According to [this](https://github.com/facebook/jest/issues/4032#issuecomment-315210901) jest cannot guarantee the order of the tests

Reply:
That makes sense. Good find!